test/e2e-v2/cases/mqe/mqe-cases.yaml

```diff
@@ -58,4 +58,8 @@ cases:
   - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics exec --expression="aggregate_labels(service_percentile,max)" --service-name=e2e-service-provider
     expected: expected/aggregateLabels-OP.yml
   - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics exec --expression="aggregate_labels(service_percentile,min)" --service-name=e2e-service-provider
-    expected: expected/aggregateLabels-OP.yml
\ No newline at end of file
+    expected: expected/aggregateLabels-OP.yml
+
+  # viewAsSeq-OP
+  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics exec --expression="view_as_seq(not_existing,service_sla)" --service-name=e2e-service-provider
```

Comment:
What would be the result if both given metrics are not existing?

Reply:
It will return the value from the first expression.